support/oidc-discovery-provider/main.go

```diff
@@ -39,13 +39,17 @@ func run(configPath string) error {
 	}
 	defer log.Close()
 
+	if config.Domain != "" {
+		log.Warn("`domain` configuration is deprecated, use domains configuration instead.")
```

Comment:
SPIRE uses documentation refers to these as "configurables" :)
```suggestion
		log.Warn("The `domain` configurable is deprecated and will be removed in a future release; use `domains` instead.")
```